app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/registration/select_entity_type.scala.html

```diff
@@ -29,10 +30,16 @@
 )
 @(form: Form[EntityType], backLink: Call)(implicit request:Request[_], messages:Messages, appConfig: ViewConfig)
 
+@toMessage(entityType: EntityType) = @{
+  entityType match {
+    case EntityType.Individual => Html(messages("entityType.individual"))
+    case EntityType.Trust      => Html(messages("entityType.trust"))
+  }
+}
 @key = @{"entityType"}
 @title = @{messages("entityType.title")}
 @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
-@options = @{List(Html(messages("entityType.individual")), Html(messages("entityType.trust")))}
+@options = @{List(EntityType.Individual, EntityType.Trust).map(toMessage)}
```

Comment:
extending `EntityType` with `Product with Serializable` enables us to write this without having to give the type to `List` as in `List[EntityType](...)` since we've got wart remover giving compile time errors when it infers a `Product` type and this:

https://nrinaudo.github.io/scala-best-practices/adts/product_with_serializable.html